addons/product/migrations/9.0.1.2/post-migration.py

```diff
@@ -276,6 +277,79 @@ def update_product_supplierinfo(env):
             AND rc.currency_id != pp.currency_id """)
 
 
+def migrate_purchase_pricelists(env):
+    """ Create product.supplierinfo records for "simple" pricelist items,
+    those are items based on a field (not other pricelist or supplier items)"""
+    env.cr.execute("select id, field from product_price_type")
```

Comment:
Considering that it's about purchase pricelists, I think here we should filter field "standard_price" only:
```suggestion
    env.cr.execute("select id, field from product_price_type where field = 'standard_price'")
```